libs/@blockprotocol/graph/rollup.config.js

```diff
@@ -0,0 +1,43 @@
+import commonjs from "@rollup/plugin-commonjs";
+import json from "@rollup/plugin-json";
+import typescript from "@rollup/plugin-typescript";
+
+const production = !process.env.ROLLUP_WATCH;
+
+const outDir = (fmt) => `dist/${fmt}`;
+
+const rolls = (fmt, input) => ({
+  input,
+  output: {
+    dir: outDir(fmt),
+    format: fmt,
+    entryFileNames: `[name].${fmt === "cjs" ? "cjs" : "js"}`,
+    name: "graph",
+    sourcemap: !production,
+  },
+  plugins: [
+    typescript({
+      declaration: true,
+      declarationDir: outDir(fmt),
+      outDir: outDir(fmt),
+      rootDir: "src",
+      sourceMap: !production,
+      inlineSources: !production,
```

Comment:
I don't know if we were always bundling sourceMaps before, just flagging this may have changed things